tests/conftest.py

```diff
@@ -4,28 +4,36 @@
 import pytest
 import xarray as xr
 
-from wildfire.goes.sequence import GoesSequence
+from wildfire import goes
 
 
 @pytest.fixture()
 def reflective_band():
-    return xr.open_dataset(
+    return xr.load_dataset(
```

Comment:
Why did this make it faster @camenpihor ?

Reply:
It doesn't! Its basically just the same as 

```
ds = xr.open_dataset()
ds.load()
```

It just loads the dataset into memory immediately, instead of the usual "lazy loading" that xarray uses (leads to weird problems)